samples/sensor/die_temp_polling/boards/stm32h573i_dk.overlay

```diff
@@ -0,0 +1,8 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+&die_temp {
+  	status = "okay";
+};
```

Comment:
Could you move that directly in the stm32h573i_dk.dts?
Ideally create a separate commit to have one commit for the dtsi and another for the dts, but not required.